usaspending_api/references/tests/test_load_psc.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from django.core.management import call_command
+from datetime import datetime
+from usaspending_api.references.models import PSC
+
+
+@pytest.mark.django_db
+def test_load_psc():
+    """
+    Test to ensure the ingested data size is correct
+    Test to make sure previously missing information has been added
+
+    """
+
+    call_command("load_psc")
```

Comment:
Since the command isn't a nuke and pave, does this really test anything? What if it fails to update this row?

Reply:
the test database starts empty, so it tests if it loads the data in with the correct fields.